src/proto/nc_redis.c

```diff
@@ -1920,6 +1920,11 @@ redis_parse_rsp(struct msg *r)
 
                     break;
                 }
+
+                if (ch == CR) {
```

Comment:
An alternative from 3e55462c0e5d9d95a88e099a777b7b3ebd373065 in a fork

```
-                state = SW_RUNTO_CRLF;
+                if (ch == '\r') {
+                    state = SW_ALMOST_DONE;
+                } else {
+                    // Read remaining characters until '\r'
+                    state = SW_RUNTO_CRLF;
+                }
             }
```

Both approaches work.